test/KeyRegistry/KeyRegistry.symbolic.t.sol

```diff
@@ -170,95 +179,7 @@ contract KeyRegistrySymTest is SymTest, Test {
-        vm.assume(selector != keyRegistry.VERSION.selector);
-        vm.assume(selector != keyRegistry.eip712Domain.selector);
-        vm.assume(selector != keyRegistry.gatewayFrozen.selector);
-        vm.assume(selector != keyRegistry.gracePeriod.selector);
-        vm.assume(selector != keyRegistry.guardians.selector);
-        vm.assume(selector != keyRegistry.idRegistry.selector);
-        vm.assume(selector != keyRegistry.isMigrated.selector);
-        vm.assume(selector != keyRegistry.keyAt.selector);
-        vm.assume(selector != keyRegistry.keyDataOf.selector);
-        vm.assume(selector != keyRegistry.keyGateway.selector);
-        vm.assume(selector != keyRegistry.keys.selector);
-        vm.assume(selector != bytes4(0x1f64222f)); // keysOf
-        vm.assume(selector != bytes4(0xf27995e3)); // keysOf paged
-        vm.assume(selector != keyRegistry.maxKeysPerFid.selector);
-        vm.assume(selector != keyRegistry.migratedAt.selector);
-        vm.assume(selector != keyRegistry.migrator.selector);
-        vm.assume(selector != keyRegistry.nonces.selector);
-        vm.assume(selector != keyRegistry.owner.selector);
-        vm.assume(selector != keyRegistry.paused.selector);
-        vm.assume(selector != keyRegistry.pendingOwner.selector);
-        vm.assume(selector != keyRegistry.totalKeys.selector);
-        vm.assume(selector != keyRegistry.validators.selector);
-
-        // Create symbolic values to be included in calldata
-        uint256 fid = svm.createUint256("fid");
-        uint32 keyType = uint32(svm.createUint(32, "keyType"));
-        bytes memory key = svm.createBytes(32, "key");
-        uint8 metadataType = uint8(svm.createUint(8, "metadataType"));
-        bytes memory metadata = svm.createBytes(32, "metadata");
-        uint256 deadline = svm.createUint256("deadline");
-        bytes memory sig = svm.createBytes(65, "sig");
-
-        // Halmos requires symbolic dynamic arrays to be given with a specific size.
-        // In this test, we provide arrays with length 2.
-        IKeyRegistry.BulkAddData[] memory addData = new IKeyRegistry.BulkAddData[](2);
-        IKeyRegistry.BulkResetData[] memory resetData = new IKeyRegistry.BulkResetData[](2);
-
-        // New scope, stack workaround.
-        {
-            bytes[][] memory fidKeys = new bytes[][](2);
-            fidKeys[0] = new bytes[](1);
-            fidKeys[0][0] = key;
-
-            bytes memory key2 = svm.createBytes(32, "key2");
-            fidKeys[1] = new bytes[](1);
-            fidKeys[1][0] = key2;
-
-            uint256 fid2 = svm.createUint256("fid2");
-
-            IKeyRegistry.BulkAddKey[] memory keyData1 = new IKeyRegistry.BulkAddKey[](1);
-            IKeyRegistry.BulkAddKey[] memory keyData2 = new IKeyRegistry.BulkAddKey[](1);
-            keyData1[0] = IKeyRegistry.BulkAddKey({key: key, metadata: ""});
-            keyData2[0] = IKeyRegistry.BulkAddKey({key: key2, metadata: ""});
```

Comment:
the createCalldata() cheatcode generates bytes of sizes 0, 32, 65, and 1024 for each `bytes` parameter, as specified by the `--default-bytes-lengths 0,32,1024,65` flag set earlier.